tests/cephfs/cephfs_pool_pg_size_change.py

```
@@ -0,0 +1,213 @@
+        sudo=True, cmd="ceph pg stat | awk '{{print $1 }}'"
+    )
+    total_pgs = out.read().decode()
+    while datetime.datetime.now() - starttime <= timeout:
+        out, rc = client[0].exec_command(
+            sudo=True, cmd="ceph pg stat | awk '{{print $3 }}'"
+        )
+        active_clean_pgs = out.read().decode()
+        out, rc = client[0].exec_command(
+            sudo=True, cmd="ceph pg stat | awk '{{print $4 }}'"
+        )
+        output = out.read().decode()
+        if active_clean_pgs == total_pgs and "active+clean" in output:
+            log.info("All pgs are active+clean")
+            return 0
+    log.error("pgs are not active+clean")
+    return 1
+
+
+def run(ceph_cluster, **kw):
+    """
+    Pre-requisites :
+    1. Create cephfs volume
+       creats fs volume create <vol_name>
+    2. Disable pg_autoscale_mode for cephfs pools if set
+       ceph osd pool set cephfs_data pg_autoscale_mode off
+       ceph osd pool set cephfs_metadata pg_autoscale_mode off
+    3. Configure 2 clients with Fuse client and another 1 client with kernel client
+
+    Test operation:
+    1. Run IO's on both clients
+    2. Verify there are no "heartbeat_map" timeout issue in logs
+    3. Fill up cluster upto 20%
+    4. Change cephfs data and metadata pool pg_num and pgp_num to existing size "-1" with client IO running
+    5. Wait for cluster to come to active + clean state , while running IO's
+    6. Write some more data to the cluster
+    7. Change cephfs data and metadata pool pg_num and pgp_num to existing size "+1" with client IO running
+    8. Wait for cluster to come to active + clean state , while running IO's
+
+    Clean-up:
+    1. Remove all the data in Cephfs file system
+    2. Remove all the cephfs mounts
+    3. Reset pg_autoscale_mode for cephfs pools to on
+    """
+    try:
+        tc = "CEPH-83574596"
+        log.info("Running cephfs %s test case" % (tc))
+
+        config = kw.get("config")
```

As we need `config`, fail fast by doing `config = kw["config"]`